docs/statistics/regression.md

```diff
@@ -0,0 +1,422 @@
+    y2 = [2.00, 2.41, 2.69, 3.41, 3.43, 3.82, 4.18, 4.36, 4.45, 4.75, 4.69]
+
+    df = pd.DataFrame({'x1': x1, 'y1': y1, 'x2': x2, 'y2': y2})
+
+
+    # Create Plotly Express figure
+    fig = px.scatter(df, x='x1', y='y1')
+
+    # Adjust the plot
+    fig.update_layout(
+        xaxis_title_text='x1',
+        yaxis_title_text='y1',
+        title=dict(
+                text='<b><span style="font-size: 10pt">Dataset 1</span></b>',
+            ),
+    )
+
+    fig.update_layout(yaxis_range=[0,6])
+
+    # Show the plot
+    fig.show()
+    fig.write_html("outputpic/regression_scatter_unscale1.html", full_html=False, include_plotlyjs='cdn')
+
+
+    # Create Plotly Express figure
+    fig2 = px.scatter(df, x='x2', y='y2')
+
+    # Adjust the plot
+    fig2.update_layout(
+        xaxis_title_text='x2',
+        yaxis_title_text='y2',
+        title=dict(
+                text='<b><span style="font-size: 10pt">Dataset 2</span></b>',
+            ),
+    )
+
+    fig2.update_layout(yaxis_range=[0,6])
+
+    # Show the plot
+    fig2.show()
+    ```
+
+Proper scaling reveals:
+
+1. For every \(X\) value, the corresponding \(Y\) value in Sample 2 is consistently larger than in Sample 1.
+2. The change in \(Y\) is more significant in Sample 2 compared to Sample 1 when \(X\) changes.
+
+
+This phenomenon occurs because we intuitively focus on the **overall picture** and draw a mental line through the points. The question then arises: how do we determine this line? This leads us into the core of linear regression, where we aim to model the relationship between variables and make informed predictions.
+
+<div class="grid cards" markdown>
+
+-   
+
+    <iframe src="/assets/statistics/regression_scatter_scale_fit1.html" width="100%" height="400px"></iframe>
+
+-  
+
+    <iframe src="/assets/statistics/regression_scatter_scale_fit2.html" width="100%" height="400px"></iframe>
+
+</div>
+
+??? code "Code"
+    ``` py
+    import pandas as pd
+    import numpy as np
+    from sklearn.linear_model import LinearRegression
+    import plotly.express as px
+
+    x1 = [0.00, 0.12, 0.18, 0.26, 0.40, 0.51, 0.60, 0.68, 0.80, 0.80, 0.99]
+    y1 = [0.23, 0.31, 0.49, 1.11, 1.03, 1.32, 1.58, 1.66, 1.65, 1.85, 1.69]
+
+    x2 = [0.14, 0.25, 0.18, 0.27, 0.42, 0.50, 0.62, 0.70, 0.79, 0.85, 1.00]
+    y2 = [2.00, 2.41, 2.69, 3.41, 3.43, 3.82, 4.18, 4.36, 4.45, 4.75, 4.69]
+
+    df = pd.DataFrame({'x1': x1, 'y1': y1, 'x2': x2, 'y2': y2})
+
+    # Linear Regression Sample 1
+    model1 = LinearRegression()
+    model1.fit(df[['x1']], df['y1'])
+    intercept1 = model1.intercept_
+    slope1 = model1.coef_[0]
+    df['y1_hat'] = intercept1 + slope1 * df['x1']
+
+    # Linear Regression Sample 2
+    model2 = LinearRegression()
+    model2.fit(df[['x2']], df['y2'])
+    intercept2 = model2.intercept_
+    slope2 = model2.coef_[0]
+    df['y2_hat'] = intercept2 + slope2 * df['x2']
+
+    # Create Plotly Express figure
+    fig = px.scatter(df, x='x1', y='y1')
```

Comment:
In plotly, you can directly add the regression line with `trendline="ols"` ([docs](https://plotly.com/python/linear-fits/)).
`fig = px.scatter(df, x='x1', y='y1', trendline="ols")`.

But I guess, you want to show how it's done 'manually'?